_dependencies/defaults/main.yml

```diff
@@ -26,3 +26,5 @@ reboot_on_package_upgrade: false                          # when set to true, re
 # Vulnerability scanning agents configurations
 qualys_agent_pkg: ""
 nessus_agent_pkg: ""
+
+merge_dict_vars_list: []

```

Comment:
This will never be defined in here, so perhaps should not exist here at all?  Instead, should test for it in the `when:` clause in `_dependencies/tasks/main.yml`

Reply:
`when: merge_dict_vars_list is defined and merge_dict_vars_list | length > 0`